tests/fixture/tmpnet/network_test.go

```diff
@@ -16,7 +19,16 @@ func TestNetworkSerialization(t *testing.T) {
 	tmpDir := t.TempDir()
 
 	network := NewDefaultNetwork("testnet")
-	require.NoError(network.EnsureDefaultConfig(&bytes.Buffer{}, "/path/to/avalanche/go", ""))
+	// TODO(marun) Remove when zap.NewNop() is possible
+	log := logging.NewLogger(
+		"",
+		logging.NewWrappedCore(
+			logging.Verbo,
+			os.Stdout,
+			zapcore.NewConsoleEncoder(zapcore.EncoderConfig{}),
+		),
+	)
```

Comment:
If we want to use `zap.NewNop()` - should we be using `logging.NoLog` right now?